webClient/src/app/shared/dialog/confirm-action/confirm-action-component.scss

```diff
@@ -0,0 +1,24 @@
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+  
+  SPDX-License-Identifier: EPL-2.0
+  
+  Copyright Contributors to the Zowe Project.
+*/
+
+mat-dialog-actions {
```

Comment:
This is already in `zlux-editor\webClient\src\styles.scss` so does this do anything?

Reply:
Addressed it and removed the file